pkg/database/database.go

```diff
@@ -310,6 +317,11 @@ func NewDB(dbName string, multidbHandler sql.MultiDBHandler, op *Options, log lo
 		return nil, err
 	}
 
+	dbi.documentEngine, err = document.NewEngine(dbi.st, sql.DefaultOptions().WithPrefix([]byte{DocumentPrefix}))
```

Comment:
it should not be related to sql, at least externally

sql.DefaultOptions()-> document.DefaultOptions()